presto-spi/src/main/java/com/facebook/presto/spi/resourceGroups/ResourceGroupInfo.java

```diff
@@ -142,7 +156,6 @@ public ResourceGroupInfo createSingleNodeInfo()
                 getNumEligibleSubGroups(),
                 getMemoryUsage(),
                 getNumAggregatedRunningQueries(),
-                getNumAggregatedQueuedQueries(),
-                emptyList());
+                getNumAggregatedQueuedQueries());
```

Comment:
I added this because the newly added test complains. I think because originally I have the constructor with only one parameter without a json property annotation.